### PR TITLE
dep: init at 0.3.1

### PR DESCRIPTION
**Pull request by carlsverre**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/dep/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      mkdir -p $out/bin
+      cp $src $out/bin/dep
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Go dependency management tool";
+    homepage = https://github.com/golang/dep;
+    maintainers = [ maintainers.carlsverre ];
+    platforms = platforms.linux;
+  };
```
Comment (vyp) [inline pkgs/development/tools/dep/default.nix]: Add the license: `license = licenses.bsd3;`

**Review by vyp — CHANGES_REQUESTED** [review]

**Comment by orivej** [comment]
We should compile it from source. It doesn't even have any dependencies, which makes it trivial to write the expression by hand, without `go2nix`'

**Comment by carlsverre** [comment]
Switching it to be from source now.

**Comment by orivej** [comment]
Thanks!

**Pull request merged**